R/plume-handler.R

```diff
@@ -128,12 +124,16 @@ PlumeHandler <- R6Class(
       private$plume <- nest(out, !!col := any_of(col))
     },
 
-    get_nestables = function() {
-      nestables <- private$get_names(private$plume_keys$nestable)
-      if (!private$crt) {
-        return(nestables)
+    get_vars = function() {
+      nestables <- private$get_names("nestable")
+      if (private$crt) {
+        nestables <- nestables[names(nestables) != "role"]
```

Comment:
This will return an empty vector since `$get_names()` now drops keys by default.